### PR TITLE
[RuntimeFilter](exec) support min max runtime filter and do refactor

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/AbstractPhysicalPlan.java [hunk 1]
```diff
@@ -135,10 +135,7 @@ public boolean pushDownRuntimeFilter(CascadesContext context, IdGenerator<Runtim
         } else {
             // null safe equal runtime filter only support bloom filter
             EqualPredicate eq = (EqualPredicate) builderNode.getHashJoinConjuncts().get(exprOrder);
-            if (eq instanceof NullSafeEqual && type == TRuntimeFilterType.IN_OR_BLOOM) {
-                type = TRuntimeFilterType.BLOOM;
-            }
-            if (eq instanceof NullSafeEqual && type != TRuntimeFilterType.BLOOM) {
+            if (eq instanceof NullSafeEqual || type == TRuntimeFilterType.BITMAP) {
```
Comment (englefly) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/AbstractPhysicalPlan.java:138]: do not need this "if"

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/AbstractPhysicalPlan.java [hunk 2]
```diff
@@ -135,10 +135,7 @@ public boolean pushDownRuntimeFilter(CascadesContext context, IdGenerator<Runtim
         } else {
             // null safe equal runtime filter only support bloom filter
```
Comment (englefly) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/AbstractPhysicalPlan.java:136]: remove this comment please

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**